config/configgrpc/configgrpc.go

```diff
@@ -110,3 +125,56 @@ func GrpcSettingsToDialOptions(settings GRPCSettings) ([]grpc.DialOption, error)
 
 	return opts, nil
 }
+
+// Config loads TLS certificates and returns a TLS Config.
+func (c TLSConfig) Config() (*tls.Config, error) {
```

Comment:
Maybe `LoadConfig`? or `ToCryptoTlsConfig`

Reply:
I like more load as it really loads it.